### DISCUSSION TITLE
Uninstall pwndbg

### DISCUSSION THREAD

**Discussion by TabKat** [pwndbg/pwndbg#1155] [General]
How to remove pwndbg?

**Comment by gsingh93** [comment]
You can remove the line that `source`s pwndbg from your `~/.gdbinit` file. `pwndbg` doesn't install itself anywhere, it's in the git folder you cloned, so you can delete that to delete the project. However, all the dependencies will still remain, you can go through the `requirements.txt` file and `setup.sh` script if you want to know which ones to remove.

**Comment by TabKat** [comment]
Thank you, @gsingh93! It worked for me.

**Comment by disconnect3d** [comment]
As of the latest dev branch, all Python dependencies will install into pwndbg/.venv so it will be even easier to remove Pwndbg and its deps completely.

Closing this since it is resolved
